typings/vscode-mssql.d.ts

```diff
@@ -489,85 +489,112 @@ declare module 'vscode-mssql' {
 		executeAndScript = 2
 	}
 
+	export interface DacDeployOptionPropertyBoolean {
```

Comment:
please add doc comments for describing what these interfaces are for

Reply:
done